src/clr-core/common/services/common-strings.default.ts

```diff
@@ -71,4 +71,9 @@ export const commonStringsDefault: ClrCommonStrings = {
   //Vertical Nav
   verticalNavToggle: 'Toggle vertical navigation',
   verticalNavGroupToggle: 'Toggle vertical navigation group',
+  // Datagrid expandable rows
+  beginningOf: 'Beginning of',
```

Comment:
`beginningOf` and `endOf` of ...? They need to be more descriptive.

Reply:
Yep, you are right about that. Will update them